link/link.go

```diff
@@ -92,11 +99,40 @@ type RawLinkOptions struct {
 	Flags uint32
 }
 
-// RawLinkInfo contains metadata on a link.
-type RawLinkInfo struct {
+// LinkInfo contains metadata on a link.
+type LinkInfo struct {
 	Type    Type
 	ID      ID
 	Program ebpf.ProgramID
+	extra   interface{}
+}
+
+// RawLinkInfo deprecated: use LinkInfo instead.
+type RawLinkInfo = LinkInfo
+
+func (r LinkInfo) ExtraRawTracepoint() *sys.RawTracepointLinkInfo {
```

Comment:
I was thinking of returning `(*RawInfo, bool)` but I guess this works too. Can you document that the functions can return nil?

Reply:
Document added.